tests/garage/envs/dm_control/test_dm_control_env.py

```diff
@@ -10,13 +12,19 @@
 
 class TestDmControlEnv(unittest.TestCase):
     @params(*dm_control.suite.ALL_TASKS)
-    def test_all_pickleable(self, domain, task):
+    def test_all_pickleable_and_step(self, domain, task):
```

Comment:
We can save some time by parameterizing this class (rather than the individual functions) and then making a single envi n `setUp()`

Reply:
I will combine it with `DmTestCase` in fixtures.py and delete this file.